tests/insurance_test.py

```diff
@@ -0,0 +1,50 @@
+import pytest
+
+import easypost
+import tests.fixture as fixture
+
+
+@pytest.mark.vcr()
+def test_insurance_create():
+    shipment = easypost.Shipment.create(**fixture.one_call_buy_shipment())
+
+    insurance = easypost.Insurance.create(
+        to_address=fixture.basic_address(),
+        from_address=fixture.basic_address(),
+        tracking_code=shipment.tracking_code,
+        carrier=fixture.usps(),
+        amount="100",
```

Comment:
Nit: You could use a fixture for this instead (see the Node PR for an example)

Reply:
A helper function is created on the top of test_insurance.py